### ISSUE TITLE
Error when no ~/.aws/config file

### ISSUE THREAD

**Issue by diasjorge** [99designs/aws-vault#233]
After doing
```
aws-vault add example
```

I entered my credentials, and got this error
```
aws-vault: error: Error adding profile: &errors.errorString{s:"No iniFile to add to"}
```

After I created the file ```touch ~/.aws/config``` The problem went away.

Perhaps it would be safe to just create the file if it doesn't exist or catch the error suggest to create the file.

Thanks

**Comment by mostafazh** [comment]
I think this has been resolved in #225 (v4.2.1). @diasjorge please confirm :grinning:

**Comment by diasjorge** [comment]
Issue resolved. Thanks :)
